sqflite/lib/src/sentry_database.dart

```diff
@@ -31,6 +33,19 @@ class SentryDatabase extends SentryDatabaseExecutor implements Database {
   static const dbSqlQueryOp = 'db.sql.query';
 
   static const _dbSqlOp = 'db.sql.transaction';
+  @internal
+  // ignore: public_member_api_docs
+  static const dbSystemKey = 'db.system';
+  @internal
+  // ignore: public_member_api_docs
+  static const dbSystem = 'sqlite';
+  @internal
+  // ignore: public_member_api_docs
+  static const dbNameKey = 'db.name';
+  @internal
+  // ignore: public_member_api_docs
+  static String? dbName;
```

Comment:
h: Is this supposed to be static? Is it not possible to run two instances of SentryDatabase in the same project pointing to different sqlite files?

Reply:
True, it would be better to couple it to the database instance, it's certainly possible to have multiple instances